### PR TITLE
salesforce doc and images

### PR DESCRIPTION
**Pull request by Adorism**
## Change description

This updates the salesforce docs. The branch was initially intended to also update facebook docs but I am waiting on some answers about setup from a contact at Meta before adding that doc and will use a new branch at that point. I will want to revisit the object-type destination for salesforce after further research. 

### Development

- [X] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/docs/integrations/grouparoo-salesforce.mdx [hunk 1]
```diff
@@ -1,193 +1,270 @@
...
+  </tablebody>
+</table>
 
-This is the simplest form of [Generator](/docs/getting-started/product-concepts#generator) for Destinations:
+## Setup
 
-```
-$ grouparoo generate salesforce:destination salesforce_destination --parent my_salesforce_app
-```
+You will need a Salesforce account in order to start syncing data to Salesforce. Your username and email will be part of setting up the App to connect to Salesforce. You’ll also need to be sure to have a security token set up before you proceed with configuration. .
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-salesforce.mdx]:
```suggestion
You will need a Salesforce account in order to start syncing data to Salesforce. Your username and email will be part of setting up the App to connect to Salesforce. You’ll also need to be sure to have a security token set up before you proceed with configuration.
```

File: pages/docs/integrations/grouparoo-salesforce.mdx [hunk 2]
```diff
@@ -1,193 +1,270 @@
...
-        "High Value Customers": "highValueCustomers",
-      },
-    },
-  ];
-};
-```
+### Sync Modes
 
-For more information on what all these properties mean and how to configure them, see [Configuring your Destination](/docs/config/code-config/destinations#configuring-your-destination).
+- **Sync**: Add, update, and remove Salesforce Accounts as needed.This option looks for new records, changes to existing records, and deletions.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-salesforce.mdx]:
```suggestion
- **Sync**: Add, update, and remove Salesforce Accounts as needed. This option looks for new records, changes to existing records, and deletions.
```

File: pages/docs/integrations/grouparoo-salesforce.mdx [hunk 3]
```diff
@@ -1,193 +1,270 @@
...
+
+This Destination enables you to sync your Grouparoo Records as Objects in Salesforce.
+
+Companies use Grouparoo to keep their Salesforce Objects up-to-date in order to manage client relations effectively.
+
+To [create a Destination](https://www.grouparoo.com/docs/config/destinations), you make choices about what data to sync.
+
+### Sync Modes
+
+- **Sync**: Add, update, and remove Salesforce Objects as needed.This option looks for new records, changes to existing records, and deletions.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-salesforce.mdx]:
```suggestion
- **Sync**: Add, update, and remove Salesforce Objects as needed. This option looks for new records, changes to existing records, and deletions.
```

File: pages/docs/integrations/grouparoo-salesforce.mdx [hunk 4]
```diff
@@ -1,193 +1,270 @@
...
+### Destination Options
 
-```js
-mapping: {
-  Email: "email",
-  FistName: "firstName",
-  "custom_lifetime_value__c": "ltv",
-},
-```
+Some types of Destinations require further clarification about how data should sync. In the case of Salesforce Export Object Destinations, there are two required options to configure before moving on to mapping Record Data: **Record Object** and **Record Match Field.** This tells Grouparoo which Account field will be used to match up Salesforce Contacts with Grouparoo Records with a unique identifier. Depending on the Objects you have set up in Salesforce, there are other optional fields to use in setting up your Destination Options
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-salesforce.mdx:262]: Curious why we don't have a screenshot for this config like we do for the others

Comment (Adorism) [reply pages/docs/integrations/grouparoo-salesforce.mdx:262]: This is the part I mentioned that needs some clarification about how the object type destination works with options. I might be able to get a screenshot in anyway. It shouldn't change drastically based on how the options are set up. I'll try it out.

**Review by teallarson — APPROVED** [review]

**Review by edmundito — APPROVED** [review]

**Pull request merged**